src/Microsoft.PowerShell.SHiPS/SHiPSProvider.cs

```diff
@@ -15,7 +15,8 @@ namespace Microsoft.PowerShell.SHiPS
     /// allows for stateless namespace navigation of a system you are modeling.
     /// </summary>
     [CmdletProvider(SHiPSProvider.ProviderName, ProviderCapabilities.Filter | ProviderCapabilities.ShouldProcess)]
-    public class SHiPSProvider : CodeOwls.PowerShell.Provider.Provider
+    public class SHiPSProvider : CodeOwls.PowerShell.Provider.Provider,
+        IContentCmdletProvider
```

Comment:
>IContentCmdletProvider [](start = 8, length = 22)

The way that the SHiPS communicates to PowerShell engine is going through P2F. We do not directly inherit from PowerShell interfaces.  P2F has defined a whole new set of interfaces, for example, IGetItemContent for get-content. We need to inherit from IGetItemContent, just for consistently design.

On the other hand, it looks pretty clean by directly inherits from IContentCmdletProvider for displaying error messages. My only concerns is when we have time in the future to implement get-content but do not have time for New-Item, for example.  In that case, we need to  write implementation for IGetItemContent. In that case, which GetContentWriter() will be called. Compile error or runtime error?